evm/src/executor/abi/mod.rs

```diff
@@ -129,6 +129,22 @@ ethers::contract::abigen!(
             parseJson(string, string)(bytes)
             parseJson(string)(bytes)
             allowCheatcodes(address)
+            serializeBool(string,string,bool)(string)
+            serializeBool(string,string,bool[])(string)
+            serializeUint(string,string,uint256)(string)
+            serializeUint(string,string,uint256[])(string)
+            serializeInt(string,string,int256)(string)
+            serializeInt(string,string,int256[])(string)
+            serializeAddress(string,string,address)(string)
```

Comment:
what's the first argument here?
the json path?

Reply:
object key, value key, value